research/delf/delf/python/pooling_layers/pooling.py

```diff
@@ -19,19 +19,16 @@
 
 class MAC(tf.keras.layers.Layer):
   """Global max pooling (MAC) layer.
-
```

Comment:
there are many blank-line removals in this file, can you please revert those?

Reply:
Thank you for pointing this out, must have missed some code formatting issue. Should be fixed now